libvmi/driver/kvm/kvm_private.h

```diff
@@ -33,15 +33,28 @@
 
 #include "private.h"
 #include "libvirt_wrapper.h"
+#ifndef KVM_LEGACY_DRIVER
+# include "driver/kvm/include/kvmi/libkvmi.h"
```

Comment:
You probably should just do `#include "include/kvmi/libkvmi.h"` instead of specifying the full path from the top.